### PR TITLE
Email updates

### PR DESCRIPTION
**Pull request by csokol**
With this PR, users can choose to receive updates of all new questions (this will be very useful for us to keep track of meta.mamute.org). Can you take a look, @leocwolter?

### CODE REVIEW

File: src/main/resources/templates/new_question_notification.ftl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<html>
+<body style="font-family:arial, sans-serif; font-size:12px;">
+	<img style="margin-bottom:30px;" src="${logoUrl}" />
```
Comment (leocwolter) [inline src/main/resources/templates/new_question_notification.ftl]: Identation?

File: src/main/java/org/mamute/mail/NewQuestionMailer.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+				.with("question", question)
+				.with("bundle", bundle)
+				.with("questionLink", questionLink)
+				.with("logoUrl", emailLogo);
+		for (User user : subscribed) {
+			boolean notSameAuthor = !user.equals(question.getAuthor());
+			if (notSameAuthor) {
+				Email email = template.to(user.getName(), user.getEmail());
+				mailer.asyncSend(email);
+			}
```
Comment (leocwolter) [inline src/main/java/org/mamute/mail/NewQuestionMailer.java]: Maybe a test for this?

**Comment by leocwolter** [comment]
Nice @csokol, can u deploy this at meta.mamute?

File: src/main/java/org/mamute/mail/NewQuestionMailer.java (hunk 2 above)
Comment (csokol) [inline src/main/java/org/mamute/mail/NewQuestionMailer.java]: it's really hard to test it, I'd have to create a bunch of mocks. I can also create an guava's predicate and use it here with Collections2.filter and unit test the predicate, what do you think? Is that too much just for a unit test?

Comment (leocwolter) [inline src/main/java/org/mamute/mail/NewQuestionMailer.java]: I would just test if mailer.asyncSend is never called if the user is the author

Comment (csokol) [inline src/main/java/org/mamute/mail/NewQuestionMailer.java]: done! https://github.com/caelum/mamute/commit/dd6104ba5dd1558d8561169a3b6b3f089523244f

Comment (csokol) [inline src/main/java/org/mamute/mail/NewQuestionMailer.java]: done: https://github.com/caelum/mamute/commit/fec99f9e9a8598b1aae48d71d28c37dca1c6998f

Comment (cpfeiffer) [inline src/main/java/org/mamute/mail/NewQuestionMailer.java]: I am out of the office until 11.05.2015.

In dringenden F�llen, kontaktieren Sie bitte
Erwin<punkt>Tratar<at>gebit<punkt>de oder
Tom<punkt>Krauss<at>gebit<punkt>de.


Note: This is an automated response to your message  "Re: [mamute] Email
updates (#156)" sent on 10.05.2015 01:43:55.

This is the only notification you will receive while this person is away.

**Pull request merged**